frontend/README.md

```diff
@@ -0,0 +1,40 @@
+# Frontend code challenge
+
+'allygator shuttle', door2door's mobility service that operates in Berlin, provides a web app that allows users to book pooled rides and track their progress. In order to enable users to be able to track the progress of their ride after they book it, door2door needs you to provide a solution which:
+
+1. shows the user the live location of their assigned vehicle on a map
+2. shows the ride's current status (waiting, in the vehicle, dropped off) in a UI component
+3. shows the ride's pickup and dropoff addresses in a UI component
+
+door2door has provided a WebSockets endpoint that will provide all of this information. It will send different events simulating the various states of a ridepooling ride. Your solution should visualise the information that is sent from this endpoint.
```

Comment:
```suggestion
door2door has provided a WebSockets endpoint that will stream all of this information. It will send different events simulating the various states of a ridepooling ride. Your solution should visualise the information that is sent from this endpoint.
```

To avoid the duplicate use of `provide`